apps/ad-campaigns/README.md

```diff
@@ -0,0 +1,5 @@
+# Ad Campaigns
```

Comment:
Nit: should we consider another folder for reusable pieces of UI? Right now, the two folders in apps are both self-contained apps in themselves. Or consider renaming "apps" => "templates"?

Reply:
yeah it feels weird with the ads. They don't fit nicely with the "apps" and they don't belong in "examples". 
Maybe `partials`, `components` or `ui-elements`?

Reply:
I like to keep the `apps` folder for the news-sites and todomvc apps though and rather find another home for the ads

Reply:
`ui-widgets` - a collection of small independent apps that can get embedded into a workload. 

These ui-widgets are functional standalone apps, that might not be beneficial to run in a benchmark by itself, but might be helpful to a benchmark, when embedding into another workload.

Reply:
Works for me!

Reply:
yay!